rune/libcontainer/specconv/spec_linux.go

```diff
@@ -339,6 +339,19 @@ func createEnclaveConfig(spec *specs.Spec, config *configs.Config) {
 	}
 }
 
+// Determine whether the file is a character device
+func IsChrDev(path string) (bool) {
+	FileAttr, err := os.Stat(path)
+	if err == nil {
+		mode := FileAttr.Mode()
+		if (mode & os.ModeCharDevice) != 0x0 {
```

Comment:
You need to check whether it is a linux misc device with the major device number 10.